src/build/render/components/wrapper/wrapper.js

```diff
@@ -53,15 +53,15 @@ const langNames = {
   es: "Español"
 };
 
-const wrapper = (ctx, headings, thanks, metaboxProps, body) => {
+const wrapper = (ctx, headings, thanks, metaboxProps, body, bodyPlaintext) => {
   const {page, pageIndex, lang, buildOpts: {baseUrl}} = ctx;
   const localize = localizations(lang);
   const editPageUrl = `${REPO_URL}/edit/master/src/content${page.pageId}/readme${lang == "en" ? "" : "_" + lang}.md`;
   const imgAbsoluteUrl = page.img ?
     `${baseUrl}${page.localizedPaths[lang]}/${page.img}` :
     `${baseUrl}/assets/librarian.png`;
 
-  const plaintextPreview = page._md ? `${renderMarkdown(page._md, pageIndex, true).substring(0, PREVIEW_LENGTH_CHARS)}...` : "";
+  const plaintextPreview = bodyPlaintext ? `${bodyPlaintext.substring(0, PREVIEW_LENGTH_CHARS)}...` : "";
```

Comment:
Won't this always append the ... even if the string isn't longer than `PREVIEW_LENGTH_CHARS`?

Reply:
True, I've pushed a fix which also adds HTML escaping for this in the `<head>` since we don't want special characters from the plaintext being interpreted as HTML.